assets/helpers/externalLinks.js

```diff
@@ -152,13 +176,13 @@ function getSubsLinks(
     return buildSubsUrls(
       countryGroupId,
       customPromos[campaign],
-      intCmp,
+      defaultIntcmps,
```

Comment:
Am I right in thinking this makes the `intCmp` param to this function effectively unused? As a result, will this always overwrite any `INTCMP` that the user has when they arrive on support (i.e. from dotcom)?

Reply:
Well spotted. I'm on it.